README.md

```diff
@@ -1,10 +1,15 @@
 [![Build Status](https://dev.azure.com/asottile/asottile/_apis/build/status/asottile.flake8-walrus?branchName=master)](https://dev.azure.com/asottile/asottile/_build/latest?definitionId=26&branchName=master)
 [![Azure DevOps coverage](https://img.shields.io/azure-devops/coverage/asottile/asottile/26/master.svg)](https://dev.azure.com/asottile/asottile/_build/latest?definitionId=26&branchName=master)
 
-flake8-walrus
-================
+# flake8-walrus
 
-flake8 plugin which forbids assignment expressions (the walrus operator)
+flake8 plugin which forbids assignment expressions (the walrus operator ``:=``).
+
+It defines one flake8 violation code with prefix ``ASN``:
```

Comment:
let's move this down below installation and make it a section `## flake8 codes`` and drop the text

Reply:
Done